### ISSUE TITLE
connection refused issue

### ISSUE THREAD

**Issue by SlowMonk** [amdegroot/ssd.pytorch#476]
python train.py --dataset COCO --dataset_root /media/jake/mark-4tb3/input/datasets/coco_2014/coco/annotations

idk but this stop at '        

> else:
>             print('elseA',in_channels,v)
>             conv2d = nn.Conv2d(in_channels, v, kernel_size=3, padding=1)

point its in ssd.py 

```
def vgg(cfg, i, batch_norm=False):
    print('vgg')
    layers = []
    in_channels = i
    for idx,v in enumerate(cfg):
        print(idx)
        if v == 'M':
            print('M')
            layers += [nn.MaxPool2d(kernel_size=2, stride=2)]
        elif v == 'C':
            print('C')
            layers += [nn.MaxPool2d(kernel_size=2, stride=2, ceil_mode=True)]
        else:
            print('elseA',in_channels,v)
            conv2d = nn.Conv2d(in_channels, v, kernel_size=3, padding=1)
            print('elseB')
            if batch_norm:
                print('if batch_norm')
                layers += [conv2d, nn.BatchNorm2d(v), nn.ReLU(inplace=True)]
            else:
                print('if elseC')
                layers += [conv2d, nn.ReLU(inplace=True)]
            in_channels = v
    print('state_pool5')
    pool5 = nn.MaxPool2d(kernel_size=3, stride=1, padding=1)
    conv6 = nn.Conv2d(512, 1024, kernel_size=3, padding=6, dilation=6)
    conv7 = nn.Conv2d(1024, 1024, kernel_size=1)
    layers += [pool5, conv6,
               nn.ReLU(inplace=True), conv7, nn.ReLU(inplace=True)]
    print('return layers')
    return layers
```

**Comment by AnhPC03** [comment]
Run visdom before training, you won't get connection refuse error.
```python -m visdom.server```
then **open other tab, must always run visdom during training**
```python train.py ...```
